Buddies/Activities/ActivityTableVC.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  ActivityTableVC.swift
```

Comment:
@lukemeierj what does this file do?

Reply:
I think eventually it will be the table we use to look at the activities listed under a Topic.  Possibly subclass MyActivities and Discover from it.

For now, it was just because I needed to be more specific than `UITableViewController` in a prepare statement